workspaces/util/src/__tests__/squashWhitespace-test.ts

```diff
@@ -0,0 +1,35 @@
+import { describe, expect, it } from "@jest/globals";
+
+import { squashWhitespace } from "../squashWhitespace";
+
+describe("squashWhitespace", () => {
+  it("removes whitespaces between words if it exists", () => {
+    expect(squashWhitespace("hello   world")).toBe("hello world");
+  });
+
+  it("removes whitespaces at the end of the string", () => {
+    expect(squashWhitespace("hello world        ")).toBe("hello world");
+  });
+
+  it("removes whitespaces at the beginginng of the string", () => {
+    expect(squashWhitespace("         hello world")).toBe("hello world");
+  });
```

Comment:
To echo @elimanzo it would be nice to have some tests that use other kinds of whitespace besides space. It would be nice to also test newlines (\n), tabs (\t), and even some of the more rare Unicode characters that are considered whitespace: https://en.wikipedia.org/wiki/Whitespace_character